src/ui/common/src/components/pages/workflow/id/index.tsx

```diff
@@ -44,6 +44,8 @@ import ReactFlowCanvas from '../../../workflows/ReactFlowCanvas';
 import WorkflowHeader from '../../../workflows/workflowHeader';
 import { LayoutProps } from '../../types';
 
+export const WorkflowPageContentId = 'workflow-page-main';
```

Comment:
what do we need this id for? just curious.

Reply:
We're using this ID to load the width of the container element for the workflow ID page.